src/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 include_directories(${CMAKE_CURRENT_LIST_DIR})
 
 set(CMAKE_BUILD_WITH_INSTALL_RPATH TRUE)
-if(LINUX)
+if(UNIX)
```

Comment:
Change its order because `UNIX` includes `APPLE`

```cmake
if(APPLE)
  set(CMAKE_INSTALL_RPATH "@loader_path")
elseif(UNIX)
  set(CMAKE_INSTALL_RPATH "$ORIGIN")
endif()
```

Reply:
2b1dbf4dfb4eaac569b582471f48fac55eef01bf